src/App.jsx

```diff
@@ -2,43 +1,31 @@
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, setactiveTabId] = useState(tabs[0].id);
+  const onTabSelected = id => {
+    setactiveTabId(id);
```

Comment:
Ensure that the `Tabs` component is used correctly, as per the task requirements. The `Tabs` component should render a list of tabs, and the active tab should be highlighted. Make sure that the `onTabSelected` function is only called when a non-active tab is selected.